packages/351elec/config/distribution/configs/distribution.conf

```diff
@@ -152,3 +152,153 @@ audio.device=headphone
 audio.volume=100
 ## Device Overclock
 overclock=0
+
+# ------------ I - EMULATORS CHOICES ----------- #
+## You can override the global configuration here
+## Here is the snes example
+#snes.shaders=mysnesshader.gplsp
+
+3do.maxperf=1
+3do.ratio=4/3
+3do.rgascale=1
+amiga.ratio=4/3
+amiga.rgascale=1
+amigacd32.ratio=4/3
+amigacd32.rgascale=1
+amstradcpc.rgascale=1
+arcade.maxperf=1
+atari2600.rgascale=1
+atari5200.rgascale=1
+atari7800.rgascale=1
+atari800.rgascale=1
+atarilynx.integerscale=1
+atarilynx.ratio=core
+atarist.rgascale=1
+#TODO: confirm best default for atomiswave, this is a guess
+atomiswave.ratio=4/3
+atomiswave.rgascale=1
+cps1.rgascale=1
+cps2.rgascale=1
+cps3.rgascale=1
+colecovision.ratio=4/3
+colecovision.rgascale=1
+c128.rgascale=1
+c16.rgascale=1
+c64.rgascale=1
+#TODO: confirm best defaults for daphne, this is a guess
+daphne.ratio=4/3
+daphne.rgascale=1
+dreamcast.maxperf=1
+dreamcast.ratio=4/3
+dreamcast.rgascale=1
+easyrpg.rgascale=1
+famicom.ratio=4/3
+famicom.rgascale=1
+fbn.ratio=core
+fbn.rgascale=1
+fds.ratio=4/3
+fds.rgascale=1
+gamegear.integerscale=1
+gb.integerscale=1
+gba.ratio=3/2
+gbah.ratio=3/2
+gbc.integerscale=1
+gbch.integerscale=1
+gbh.integerscale=1
+genesis.ratio=4/3
+genesis.rgascale=1
+genh.ratio=4/3
+genh.rgascale=1
+ggh.integerscale=1
+intellivision.ratio=4/3
+intellivision.rgascale=1
+mame.emulator=libretro
+mame.core=mame2003_plus
+mame.maxperf=1
+mame.rgascale=1
+mastersystem.rgascale=1
+megacd.ratio=4/3
+megacd.rgascale=1
+megadrive.ratio=4/3
+megadrive.rgascale=1
+megadrive-japan.ratio=4/3
+megadrive-japan.rgascale=1
+msx.ratio=4/3
+msx.rgascale=1
+msx2.ratio=4/3
+msx2.rgascale=1
+n64.maxperf=1
+n64.ratio=4/3
+n64.rgascale=1
+naomi.maxperf=1
+naomi.ratio=4/3
+naomi.rgascale=1
+neocd.ratio=4/3
+neocd.rgascale=1
+neogeo.ratio=4/3
+neogeo.rgascale=1
+nes.ratio=4/3
+nes.rgascale=1
+nesh.ratio=4/3
+nesh.rgascale=1
+ngp.integerscale=1
+ngpc.intergerscale=1
+odyssey2.ratio=4/3
+odyssey2.rgascale=1
+pc.maxperf=1
+pc.ratio=4/3
+pc.rgascale=1
+pc-9800.ratio=4/3
+pc-9800.rgascale=1
+pcengine.ratio=4/3
+pcengine.rgascale=1
+pcenginecd.ratio=4/3
+pcenginecd.rgascale=1
+pcfx.maxperf=1
+pcfx.ratio=4/3
+pcfx.rgascale=1
+pokemini.ratio=3/2
+psp.maxperf=1
+pspminis.maxperf=1
+psx.ratio=4/3
+psx.rgascale=1
+sc-3000.rgascale=1
+saturn.ratio=4/3
+saturn.rgascale=1
+sega32x.ratio=4/3
+sega32x.rgascale=1
+segacd.ratio=4/3
+segacd.rgascale=1
+sfc.ratio=4/3
+sfc.rgascale=1
+sg-1000.rgascale=1
+snes.ratio=4/3
+snes.rgascale=1
+snesh.ratio=4/3
+snesh.rgascale=1
+snesmsu1.ratio=4/3
+snesmsu1.rgascale=1
+supergrafx.ratio=4/3
+supergrafx.rgascale=1
+tg16.ratio=4/3
```

Comment:
Add integer Scale for Watara Supervision
```suggestion
supergrafx.rgascale=1
supervision.integerscale=1
tg16.ratio=4/3
```

Reply:
supervision nice 🥇